docs/book/stacks-and-components/component-guide/orchestrators/custom.md

```diff
@@ -210,6 +220,19 @@ class MyOrchestrator(ContainerizedOrchestrator):
             # each step only runs after all its upstream steps finished
             upstream_steps = step.spec.upstream_steps
 
+            # You can get the config your orchestrator like so.
+            # The config is the "static" part of your orchestrators config, defined
+            # when you register your orchestrator
+            step_config = step.config  # Defined in `MyOrchestratorConfig`
```

Comment:
```suggestion
            # You can get the config of your orchestrator like so.
            # The config is the "static" part of your orchestrators config, defined
            # when you register your orchestrator
            config = self.config
```

I would suggest not even mentioning this here, maybe in the comment of the step settings below?

Reply:
Was already moved into a hint so I guess this is resolved